lib/auth/integration/integrationv1/service_test.go

```diff
@@ -375,7 +433,10 @@ func initSvc(t *testing.T, ca types.CertAuthority, clusterName string, proxyPubl
 	})
 	require.NoError(t, err)
 
-	localResourceService, err := local.NewIntegrationsService(backend)
+	localResourceService, err := local.NewIntegrationsService(backend,
+		local.WithPreDeleteChecks(local.EASIntegrationPreDeleteCheck),
+		local.WithDeleteAllIntegrationsEnabled(true),
```

Comment:
I specifically wanted both just for this test to work 😅 DeleteAll doesn't seem to be called anywhere except potentially by the cache stuff, so I didn't think it was worth setting up the PreDeleteChecks properly in there, especially since i would have to handle paging and it seemed pointless, and we probably shouldn't actually be running PreDeleteChecks in a cache